index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+ <html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Apple's Homepage</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/8620d989f5.js"></script>
+  </head>
+  <body>
+    <header>
+          <nav class="navbar">
+            <ul>
+              <li class="list"><a href="#"><img src="img/apple-icon.png" alt="icon"></a></li>
+              <li class="list"><a href="#">Store</a></li>
+              <li class="list"><a href="#">Mac</a></li>
+              <li class="list"><a href="#">Ipod</a></li>
+              <li class="list"><a href="#">Iphone</a></li>
+              <li class="list"><a href="#">Ipad</a></li>
+              <li class="list"><a href="#">Itunes</a></li>
+              <li class="list"><a href="#">Support</a></li>
+              <li class="list">
+                <form>
+                  <span class="fa fa-search" id="search"></span>
+                  <input type="text" name="search" class="search">
+                </form>
+              </li>
+            </ul>
+          </nav>
+          
+          <div class="main">
+              <section class="heading">
+                <h1>
+                  <a href="#">What will your verse be?</a>
+                </h1>
+                <h2>
+                  <a href="#">
+                    Everyone has something to share. See how people are using
+                    <br>
+                    iPad in their unique ways to add to the world's story.
+                  </a>
+                </h2>
+              </section>
+          
+              <a class="play" href="#">
+                Watch the "Your Verse" film and explore the stories <i class="far fa-play-circle"></i>
+              </a>
+          </div>
+    </header>
+        <aside class="bottom">
+          <ul>
+            <li><img src="img/iphone5.jpg" alt="iphone5"></li>
+            <li><img src="img/iphone5c.jpg" alt="iphone5C"></li>
+            <li><img src="img/mac.jpg" alt="mac"></li>
+            <li><img src="img/mac2.jpg" alt="mac2"></li>
+          </ul>
+        </aside>
+
```

Comment:
This four boxes above the footer should be anchored. Please, work on that before merging.

Reply:
Ok, thanks.